mediachain/transactor/blockchain_follower.py

```diff
@@ -26,7 +26,7 @@ def __init__(self,
         self.stream_func = stream_func
         self.block_ref_queue = Queue()
         self.incoming_event_queue = Queue()
-        self.block_replay_queue = Queue()
+        self.block_replay_queue = LifoQueue()
```

Comment:
did you want to rename this? I know they call it a LifoQueue but come on it's a stack